includes/menu.inc

```diff
@@ -303,3 +303,42 @@ function islandora_batch_set_restart_batch_form_submit(&$form, &$form_state) {
     'ingest_set' => $id,
   ));
 }
+
+/**
+ * Page callback to delete ingested objects that belong to a set.
+ */
+function islandora_batch_delete_set_object_page_callback($id, $management = FALSE) {
+  return drupal_get_form('islandora_batch_set_delete_object_form', $id, $management);
+}
+
+/**
+ * Form building function to delete ingested objects.
+ */
+function islandora_batch_set_delete_object_form($form, &$form_state, $id, $management) {
+  form_load_include($form_state, 'inc', 'islandora_batch', 'includes/ingest.batch');
```

Comment:
Not actually used here; though, could possibly add some info about lock availability, as in the [restart batch dealio](https://github.com/Islandora/islandora_batch/pull/84/files#diff-43c46518fc6d7b2100d43fa79a36c22cR236)?